python/metadata_model.py

```diff
@@ -0,0 +1,13 @@
+from schematic.models.metadata import MetadataModel
+from schematic import CONFIG
+
+
+config = CONFIG.load_config("config.yml")
```

Comment:
I need to specify path: `config = CONFIG.load_config("schematic/config.yml")`, to make it work. Not sure if it is just me.

Reply:
Hm, is it possible that you're still installing `schematic` the traditional way by cloning the git repo and using poetry/pip to build and install it? If you're using `pip install schematicpy`, then you should technically be able to place the `config.yml` file anywhere and just point the config path in the above code to that path. Here, for the sake of simplicity I've put the config file in the root of the Data Curator App, and hence it's just `config.yml`. In your case, you probably have another `schematic` folder inside which has a `config.yml` file? Is that right?

Reply:
Yeah, I was still looking for `config.yml` in `schematic` repo. I can confirm that if I place the config file in the root of the Data Curator App, the `config.yml` file can be found. 

Do we still need to clone the `schematic` repo? On my end, the files in `schematic` cannot be found if I just use `pip install schematicpy` and not clone the `schematic` repo. 

I can confirm the `schematicpy` is installed using pip. If I do not clone the `schematic` repo, the error says:
```
FileNotFoundError: [Errno 2] No such file or directory: 'data/schema_org_schemas/HTAN.jsonld'
```

Reply:
Right, that's exactly how we want it to behave. Like you did, you need to install `schematicpy` with `pip install schematicpy` first.

In order to resolve the error you're seeing, you need to create a `data/schema_org_schemas/` directory in the root of your Data Curator App and place the `HTAN.jsonld` file in there. Just a heads up, you don't need to create it in the same way that I have. You can just as freely call it `data/schemas/` or even just `data/` and place your `HTAN.jsonld` file in there. You just need to configure it appropriately in your `config.yml` file.